.github/ISSUE_TEMPLATE/chore_template.md

```diff
@@ -1,14 +1,14 @@
 ---
 name: "Chore"
-about: "Open a Chore for minor update."
+about: "Open a chore issue for a minor update."
 title: "Update "
 labels: "type : chore"
 ---
 
 ## Why
 
-Describe the update details and why it's needed. 
- 
+Describe the update details and why it's needed.
```

Comment:
Should we change to
```suggestion
Describe the update in detail and why it is needed.
```

the same as you did in https://github.com/nimblehq/git-template/pull/20/files#diff-2c69b215001999ab79729762d7a6f0567d9dc71323961a98641c57be26bbb42fR10

Reply:
d880b24 ✅